pointmatcher/ErrorMinimizers/PointToPlane.cpp

```diff
@@ -154,7 +192,22 @@ typename PointMatcher<T>::TransformationParameters PointToPlaneErrorMinimizer<T>
 		assert(normalRef.rows() > 0);
 
 		// Compute cross product of cross = cross(reading X normalRef)
```

Comment:
This comment is repeated below, and it points at the math operation, but doesn't say what's the goal of it. Could it be rephrased in a way in which we know what's the goal, and then the next comments explain some of the steps if needed?

Reply:
Also goes to my TODO list. I'll put there reference to Francois' publication with equation numbers.